screenpear/__init__.py

```diff
@@ -28,9 +28,11 @@ def cli():
 @cli.command()
 @click.option('--src', help='')
 @click.option('--dst', help='')
-def ocr(src, dst):
+@click.option('--img_w', help='', default=-1)
```

Comment:
I don't think you need to set a default to -1, but instead just check if it's defined or not as per [click docs](https://click.palletsprojects.com/en/8.1.x/options/)